s3/pkg/datastore/yig/storage/multipart.go

```diff
@@ -256,9 +256,10 @@ func (yig *YigStorage) ListParts(ctx context.Context, multipartUpload *pb.ListPa
 			break
 		}
 		p := model.Part{
-			PartNumber: part.PartNum,
-			ETag:       part.Etag,
-			Size:       int64(part.Size),
+			PartNumber:     part.PartNum,
+			ETag:           part.Etag,
+			Size:           int64(part.Size),
+			LastModifyTime: part.UpdateTime.Unix(),
```

Comment:
Doubt..When listing the parts, the last modify time should be based upon the existing value from DB and we should not set the LastModifyTime while listing..Isn't it?

Reply:
@kumarashit yes it is. The last modify time is created when the part is uploaded successfully. And we should not set it when listing it. And in this pr, the last modify time of the part is not changed in ListParts interface, it is got from DB.